geps/gep-713.md

```diff
@@ -289,6 +457,7 @@ the same resource _and_ have an identical field specified with different values,
 precedence MUST be determined in order of the following criteria, continuing on
 ties:
 
+* Hierarchical Policies override Direct Policies.
```

Comment:
You say the opposite in site-src/references/policy-attachment.md L254

Reply:
Ah, thanks, having two places to record this stuff is not ideal. I'll fix.